ADR/0017-use-our-pipelines.md

```diff
@@ -0,0 +1,44 @@
+# 17. Use our own pipelines
+
+* Date 2023-02-10
+
+## Status
+
+Accepted
+
+Relates to [ADR 7. Change Management](0007-change-management.html)
+
+## Context
+
+The maintainers of stonesoup components need to demonstrate evidence of practices that support
+a secure software development lifecycle (for example scanning, manifesting, vulnerability detection,
+etc.)
+
+There are lots of options out there for us to use, notably github actions. However, we're building
+a ci/cd platform that is meant to support a secure software development lifecycle from the start.
+
+## Decision
+
+Use our own pipelines to build and scan Stonesoup components. Almost all of our components already
+do this today. Look for evidence in the `.tekton/` directory of their git repo.
+
+However, we have stopped short of configuring and [Application] and [Components] for Stonesoup.
+We're using the pipelines directly, but not via the Stonesoup UI. This is something we intend to
+start doing, but haven't made time to do so yet.
+
+## Consequences
```

Comment:
What of teams that are packaging/deploying Red Hat products built on CPaaS? Would this be considered out of scope for now?

Reply:
Follow up question - is there an means that we can get similar data for our products?

Reply:
I think that the only parts that would be built on CPaaS are those which are currently being productized. As an example, I assume that we are using the official gitops and pipelines products.

Is that what you were asking? If so, _those_ would be out of scope, but any integration/configuration in Stonesoup would be in scope.

Reply:
Correct - _however_ I had drafted an ADR a while back related to Pipelines where we want to deploy a ".next" version. I need to update that since the Pipelines team now thinks we have a way to make this work.